rules/rules-overridden-azure/README.md

```diff
@@ -15,8 +15,92 @@ Overriding an existing rule
 Rules written in XML or Groovy can be overridden either in XML or Groovy.
 It doesn't matter if the original rule is written in XML or Groovy, you can override it in either language.
 
+To override an existing rule, the ruleset file can be named the same as the original rule (or differently), but it MUST have the same ruleset id and the same id attribute as the original rule.
```

Comment:
How about we add a windup overrides doc link here for reference: https://windup.github.io/windup-documentation/website/docs/rules-development-guide/master.html#overriding-rules_rules-development-guide

Reply:
> To override an existing rule, the ruleset file can be named the same as the original rule (or differently), but it MUST have the same ruleset id and the same id attribute as the original rule.

I think a better way to say this is:

_To override an existing rule, the ruleset must have the same ruleset id and the same id attribute as the original rule. It may have the same filename, but that is not a requirement._

Reply:
And I agree with @KaiqianYang on adding a link to the reference documentation.

Reply:
> To override an existing rule, the ruleset file can be named the same as the original rule (or differently), but it MUST have the same ruleset id and the same id attribute as the original rule.

I think a better way to say this is:

_To override an existing rule, the ruleset must have the same ruleset id and the same id attribute as the original rule. It may have the same filename, but that is not a requirement._

And I agree with @KaiqianYang on adding a link to the reference documentation.